client/app/components/Footer/index.js

```diff
@@ -2,43 +2,23 @@ import React from 'react';
 import { Link } from 'react-router';
 import Reveal from 'react-reveal';
 
+import bulma from 'styles/bulma.scss';
 import styles from './styles.scss';
 
 import LargeLogo from './white.png';
 
 const year = new Date().getFullYear();
 
 const Footer = () => (
-  <footer className="hero is-dark is-medium">
-    <div className={styles.heroBody}>
+  <footer className={`${bulma.hero} ${bulma['is-dark']} ${bulma['is-medium']}`}>
```

Comment:
FYI my last update while I was at the airport means you can reference the bulma classnames directly as plain strings instead of needing to import the Bulma object and reference the classnames as properties. You can simply use className="hero is-dark is-medium" as an example. You'll only need string templates if using Bulma classes AND custom styles from styles.scss on the same element.

I'd rather have the classnames cleaned up before accepting this PR :)